perl-unable-to-find-blacklist

```diff
@@ -0,0 +1 @@
+recipes/perl-font-afm
```

Comment:
I meant to ask you the same thing. There are some recipes that I can run `conda build /bioconda/recipes/perl-font-afm`, and I'm fine, but running using the bioconda utils decides there are no files in the download, or possibly downloading has gone awry. I had a nice report on my work computer, and will get back to you with it tomorrow.